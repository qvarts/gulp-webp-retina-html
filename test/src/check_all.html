<!doctype html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport"
          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <title>Check all supported image types</title>
</head>
<body>
<img class="test-class" src="/img/test.jpg" alt="test" />
<img class="test-class" src="/img/test.jpeg" alt="test" />
<img class="test-class" src="/img/test.jfif" alt="test" />
<img class="test-class" src="/img/test.pjpeg" alt="test" />
<img class="test-class" src="/img/test.pjp" alt="test" />
<img class="test-class" src="/img/test.png" alt="test" />
<img class="test-class" src="/img/test.apng" alt="test" />
<img class="test-class" src="/img/test.gif" alt="test" />
<img class="test-class" src="/img/test.svg" alt="test" />
<img class="test-class" src="/img/test.webp" alt="test" />
<img class="test-class" src="/img/test.bmp" alt="test" />
<img class="test-class" src="/img/test.ico" alt="test" />
<img class="test-class" src="/img/test.cur" alt="test" />
<img class="test-class" src="/img/test.tif" alt="test" />
<img class="test-class" src="/img/test.tiff" alt="test" />
<img class="test-class" src="/img/test.avif" alt="test" />
</body>
</html>