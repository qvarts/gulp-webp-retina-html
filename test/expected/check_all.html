<!doctype html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport"
          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <title>Check all supported image types</title>
</head>
<body>
<picture>
<source srcset="/img/test.webp, /img/test.webp 1x, /img/test@2x.webp 2x" type="image/webp">
<source srcset="/img/test.jpg 1x, /img/test@2x.jpg 2x" type="image/jpeg">
<img class="test-class" src="/img/test.jpg" alt="test" />
</picture>
<picture>
<source srcset="/img/test.webp, /img/test.webp 1x, /img/test@2x.webp 2x" type="image/webp">
<source srcset="/img/test.jpeg 1x, /img/test@2x.jpeg 2x" type="image/jpeg">
<img class="test-class" src="/img/test.jpeg" alt="test" />
</picture>
<picture>
<source srcset="/img/test.webp, /img/test.webp 1x, /img/test@2x.webp 2x" type="image/webp">
<source srcset="/img/test.jfif 1x, /img/test@2x.jfif 2x" type="image/jpeg">
<img class="test-class" src="/img/test.jfif" alt="test" />
</picture>
<picture>
<source srcset="/img/test.webp, /img/test.webp 1x, /img/test@2x.webp 2x" type="image/webp">
<source srcset="/img/test.pjpeg 1x, /img/test@2x.pjpeg 2x" type="image/jpeg">
<img class="test-class" src="/img/test.pjpeg" alt="test" />
</picture>
<picture>
<source srcset="/img/test.webp, /img/test.webp 1x, /img/test@2x.webp 2x" type="image/webp">
<source srcset="/img/test.pjp 1x, /img/test@2x.pjp 2x" type="image/jpeg">
<img class="test-class" src="/img/test.pjp" alt="test" />
</picture>
<picture>
<source srcset="/img/test.webp, /img/test.webp 1x, /img/test@2x.webp 2x" type="image/webp">
<source srcset="/img/test.png 1x, /img/test@2x.png 2x" type="image/png">
<img class="test-class" src="/img/test.png" alt="test" />
</picture>
<picture>
<source srcset="/img/test.webp, /img/test.webp 1x, /img/test@2x.webp 2x" type="image/webp">
<source srcset="/img/test.apng 1x, /img/test@2x.apng 2x" type="image/apng">
<img class="test-class" src="/img/test.apng" alt="test" />
</picture>
<picture>
<source srcset="/img/test.webp, /img/test.webp 1x, /img/test@2x.webp 2x" type="image/webp">
<source srcset="/img/test.gif 1x, /img/test@2x.gif 2x" type="image/gif">
<img class="test-class" src="/img/test.gif" alt="test" />
</picture>
<picture>
<source srcset="/img/test.webp, /img/test.webp 1x, /img/test@2x.webp 2x" type="image/webp">
<source srcset="/img/test.svg 1x, /img/test@2x.svg 2x" type="image/svg+xml">
<img class="test-class" src="/img/test.svg" alt="test" />
</picture>
<picture>
<source srcset="/img/test.webp 1x, /img/test@2x.webp 2x" type="image/webp">
<img class="test-class" src="/img/test.webp" alt="test" />
</picture>
<picture>
<source srcset="/img/test.webp, /img/test.webp 1x, /img/test@2x.webp 2x" type="image/webp">
<source srcset="/img/test.bmp 1x, /img/test@2x.bmp 2x" type="image/bmp">
<img class="test-class" src="/img/test.bmp" alt="test" />
</picture>
<picture>
<source srcset="/img/test.webp, /img/test.webp 1x, /img/test@2x.webp 2x" type="image/webp">
<source srcset="/img/test.ico 1x, /img/test@2x.ico 2x" type="image/x-icon">
<img class="test-class" src="/img/test.ico" alt="test" />
</picture>
<picture>
<source srcset="/img/test.webp, /img/test.webp 1x, /img/test@2x.webp 2x" type="image/webp">
<source srcset="/img/test.cur 1x, /img/test@2x.cur 2x" type="image/x-icon">
<img class="test-class" src="/img/test.cur" alt="test" />
</picture>
<picture>
<source srcset="/img/test.webp, /img/test.webp 1x, /img/test@2x.webp 2x" type="image/webp">
<source srcset="/img/test.tif 1x, /img/test@2x.tif 2x" type="image/tiff">
<img class="test-class" src="/img/test.tif" alt="test" />
</picture>
<picture>
<source srcset="/img/test.webp, /img/test.webp 1x, /img/test@2x.webp 2x" type="image/webp">
<source srcset="/img/test.tiff 1x, /img/test@2x.tiff 2x" type="image/tiff">
<img class="test-class" src="/img/test.tiff" alt="test" />
</picture>
<picture>
<source srcset="/img/test.webp, /img/test.webp 1x, /img/test@2x.webp 2x" type="image/webp">
<source srcset="/img/test.avif 1x, /img/test@2x.avif 2x" type="image/avif">
<img class="test-class" src="/img/test.avif" alt="test" />
</picture>
</body>
</html>